targets/nuttx-stm32f4/README.md

```diff
@@ -16,6 +16,8 @@ git clone https://github.com/jerryscript-project/jerryscript.git
 git clone https://bitbucket.org/nuttx/nuttx.git
 git clone https://bitbucket.org/nuttx/apps.git
 git clone https://github.com/texane/stlink.git
+cd stlink
+git checkout 3d24377c2878fc328fb09b67cdfbda529d209417
```

Comment:
I indeed _suggest_ to limit the changes to section 5. The file path to the stlink  binary is one thing. The other is the flashing instructions URL, where I'd suggest to link to the repository only, without the `#build-from-sources` (which does not exist).